adafruit_emc2101/__init__.py

```diff
@@ -261,17 +281,23 @@ def external_temperature(self):
         temp_msb = self._ext_temp_msb
         full_tmp = (temp_msb << 8) | temp_lsb
         full_tmp >>= 5
-        full_tmp *= 0.125
-
-        return full_tmp
+        if full_tmp == emc2101_regs.TEMP_FAULT_OPENCIRCUIT or full_tmp == emc2101_regs.TEMP_FAULT_SHORT:
+            if self._sensorfault_exceptions:
+                raise ValueError("External Sensor fault")
+            else:
+                return int(full_tmp)
+        else:
+            full_tmp *= 0.125
+        return float(full_tmp)
 
     @property
     def fan_speed(self):
         """The current speed in Revolutions per Minute (RPM)"""
-
         val = self._tach_read_lsb
         val |= self._tach_read_msb << 8
-        return round(_FAN_RPM_DIVISOR / val, 2)
+        if val < 1:
+            return 0
+        return round(emc2101_regs._FAN_RPM_DIVISOR / val, 2)
```

Comment:
the divisor doesn't need to be (and so shouldn't be) public. Why does pylint have a problem with it?

Reply:
Because it's being accessed from outside of the class itself, `pylint` expects it to be public.  That's really what it means to be public vs private in Python - nothing truly is, but it more represents the concept of whether code external to itself should be accessing it.  Since that's what's happening here, it really should be considered public.

Reply:
I got it eventually and it's now changed. Thanks.